### PR TITLE
Add feedback for the RHEL guide

### PR DESCRIPTION
**Pull request by juliogonzalez**

### CODE REVIEW

File: modules/client-configuration/pages/clients-rh.adoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ systemctl restart taskomatic
 
 == Import Entitlements and CA Certificate
 
-{redhat} clients require a {redhat} certificate authority (CA) and entitlement certificate.
+{redhat} clients require a {redhat} certificate authority (CA) and entitlement certificate and entitlemente key.
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
{redhat} clients require a {redhat} certificate authority (CA) and entitlement certificate and entitlement key.
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 2]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
 It runs locally to track installed products and subscriptions.
 Clients must be registered with the subscription manager to obtain certificates.
 
+Depending on where the {redhat} system is registered, the URL’s to which the system is authorized to replicate differ.
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
{redhat} clients use a URL to replicate repositories.
The URL will change depending on where the {redhat} client is registered.
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 3]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+
+{redhat} systems can be registered to three different infrastructures:
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
{redhat} clients can be registered in three different ways:
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 4]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+
+1. {redhat} CDN (redhat.com)
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
* {redhat} content delivery network (CDN) at redhat.com
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 5]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+2. {redhat} Satellite Server
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
* {redhat} Satellite Server
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 6]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+3. RHUI (update infrastructure in the cloud)
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
* {redhat} update infrastructure (RHUI) in the cloud
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 7]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+
+This guide focuses on a system registered to {redhat} CDN (redhat.com). Register at least one system there with an authorized subscription to obtain repository content.
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
This guide covers clients registered to {redhat} CDN. 
You must have at least one system registered to the CDN, with an authorized subscription for repository content.
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 8]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+
+[IMPORTANT]
+====
+Entitlement certificates for RHUI (cloud-based systems) only authorize you to get content there.
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
Entitlement certificates for RHUI (cloud-based systems) only allow you to download content, not repository data.
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 9]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+Satellite certificates for client systems require a Satellite server and subscription: such client certificates are UNSUPPORTED with {productname} server.
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
Satellite certificates for client systems require a Satellite server and subscription.
Clients using Satellite certificates are not supported with {productname} Server.
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 10]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+====
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]: I feel like this admonition should be split into two: one about RHUI, one about Sat. Of course, then it means we have three admonitions in a row, which probably means it needs a little bit of re-thinking.

File: modules/client-configuration/pages/clients-rh.adoc [hunk 11]
```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
...
+
+[WARNING]
+====
+Entitlement certificates are embedded with expiration dates, which match the length of the support subscription.
+To avoid disruption, you will need to repeat this process at the end of every support subscription period.
+====
+
+{redhat} supplies the subscription-manager tool to manage subscription assignments. It runs locally on the client system to track installed products and subscriptions. Register to redhat.com with subscription-manager, then follow this procedure to obtain certificates.
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-rh.adoc]:
```suggestion
{redhat} supplies the subscription-manager tool to manage subscription assignments. 
It runs locally on the client system to track installed products and subscriptions. 
Register to redhat.com with subscription-manager, then follow this procedure to obtain certificates.
```

File: modules/client-configuration/pages/clients-rh.adoc (hunk 10 above)
Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-rh.adoc]: @dvosburg ?

**Comment by juliogonzalez** [comment]
Let's merge this with a squash.

File: modules/client-configuration/pages/clients-rh.adoc (hunk 10 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-rh.adoc]: I'm happy to edit this later on, anyway, so consider it non-blocking.

**Pull request merged**